src/cmap/test/event.rs

```diff
@@ -1,17 +1,14 @@
-use std::{
-    sync::{Arc, RwLock},
-    time::Duration,
-};
+use std::sync::{Arc, RwLock};
 
 use serde::{de::Unexpected, Deserialize, Deserializer, Serialize};
 
-use crate::{event::cmap::*, options::ServerAddress, runtime};
-use tokio::sync::broadcast::error::{RecvError, SendError};
+use crate::{event::cmap::*, options::ServerAddress, test::util::EventSubscriber};
+use tokio::sync::broadcast::error::SendError;
 
 #[derive(Clone, Debug)]
 pub struct EventHandler {
-    pub events: Arc<RwLock<Vec<Event>>>,
-    channel_sender: tokio::sync::broadcast::Sender<Event>,
+    pub(crate) events: Arc<RwLock<Vec<CmapEvent>>>,
```

Comment:
Another thing is that `pub` things (even in tests) can't expose `pub(crate)` things from the driver. In practice this doesn't matter for us because our tests don't get compiled in to the actual library, but it will prevent compilation from succeeding.